PROVISIONING.md

```diff
@@ -1,6 +1,6 @@
 # Discovery and Provisioning
 
-## Docs
+## Supportive Docs and alternative specs
```

Comment:
Weird capitalization here. Should be `Supportive docs and alternative spec`.

Reply:
fixed